spec/lib/search/service_spec.rb

```diff
@@ -32,6 +32,61 @@
         end
       end
     end
+
+    context 'with an offset' do
+      let(:query) { 'test' }
+
+      it 'returns the correct segment of results', :aggregate_failures do
+        VCR.use_cassette('search/offset') do
+          response = subject.results
+
+          next_offset = response.body['pagination']['next']
+          prev_offset = response.body['pagination']['previous']
+
+          expect(next_offset).to eq 20
+          expect(prev_offset).to eq nil
+        end
+      end
+
+      it 'returns the correct pagination offsets', :aggregate_failures do
+        VCR.use_cassette('search/offset_20') do
+          subject = described_class.new(query, '20')
+          response = subject.results
+
+          next_offset = response.body['pagination']['next']
+          prev_offset = response.body['pagination']['previous']
+
+          expect(next_offset).to eq 40
+          expect(prev_offset).to eq nil
```

Comment:
If the user is on the 2nd page of results, and they want to get the 1st page, in this case we'd want `previous: 0` (not `previous: nil`), so that we are consistent with how the FE requests the 1st page, vs. indicating to the FE that they currently **have** the first page.  You know what I mean?